### PR TITLE
Sending and Receiving Coins Using your Skywallet.md

### PR DESCRIPTION
**Pull request by sreekumar13**

### CODE REVIEW

File: Sending and Receiving Coins Using your Skywallet.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Sending and Receiving Coins Using your Skywallet
+
+Sending and receiving of Coins are two of the most basic features of your Skywallet.
```
Comment (jdknives) [inline Sending and Receiving Coins Using your Skywallet.md]: Can be deleted I think as it does not give anything to the user.

File: Sending and Receiving Coins Using your Skywallet.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+To send or receive Coins using your Skywallet, you have to initialize and personalize your Skywallet, details on how to initialize and personalize your Skywallet is explained in the following sections :
```
Comment (jdknives) [inline Sending and Receiving Coins Using your Skywallet.md]: ..personalize your Skywallet first. Details on ... can be found in the following chapters:

File: Sending and Receiving Coins Using your Skywallet.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+**[Initialization of your Skywallet](https://github.com/skycoin/hardware-wallet/wiki/Initialize-the-wallet-setting-up)**
+
+**[Personalization of your Skywallet](https://github.com/skycoin/hardware-wallet/wiki/Getting-to-know-the-wallet)**
+
+Once you have completed your initialization and personalization of your Skywallet using your Desktop Wallet you can see your Skywallet in the list of Wallets configured using your Desktop Wallet.
```
Comment (jdknives) [inline Sending and Receiving Coins Using your Skywallet.md]: completed the initialization

Comment (jdknives) [reply Sending and Receiving Coins Using your Skywallet.md]: Delete `using your desktop wallet` as it is explained in the sections above.

File: Sending and Receiving Coins Using your Skywallet.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+To receive Coins to your Skywallet, you need the address of your Skywallet.
+
+To get the address of the Skywallet, follow these steps :
+
+**Step 1** - Connect your Skywallet to your computer via USB.
+
+**Step 2** - Open the Desktop Wallet from your computer.
+
+**Step 3** - The window will display the list of all the Wallets configured using your Desktop Wallet, from this list you can click on your Skywallet name to get the Wallet address of your Skywallet.
```
Comment (jdknives) [inline Sending and Receiving Coins Using your Skywallet.md]: `.` instead of `,`

File: Sending and Receiving Coins Using your Skywallet.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+**Step 4** - Click on the **Press to Show** button.
+
+Once you have clicked on the "Press to Show" button, the address of the Skywallet will be displayed both by the Desktop Wallet as well as your Skywallet.
+
+**Wallets Showing Same Addresses**
+
+<screenshot and image of the same address being displayed>
+
+**NOTE - Please check whether both these addresses are same. If it is not the same, then cancel the operation and stop using both the Skywallet and Desktop Wallet. Immediately contact Support by visiting this link - [store.skycoin.net/pages/support](store.skycoin.net/pages/support), or you can send an email detailing the issue to **support@skycoin.zendesk.com**.
```
Comment (jdknives) [inline Sending and Receiving Coins Using your Skywallet.md]: the same.

Comment (jdknives) [reply Sending and Receiving Coins Using your Skywallet.md]: If they are not the same,... .

Comment (jdknives) [reply Sending and Receiving Coins Using your Skywallet.md]: Just say stop using the Skywallet.

Comment (jdknives) [reply Sending and Receiving Coins Using your Skywallet.md]: Make `or you can...` a separate sentence.

File: Sending and Receiving Coins Using your Skywallet.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+<screenshot and image of the same address being displayed>
+
+**NOTE - Please check whether both these addresses are same. If it is not the same, then cancel the operation and stop using both the Skywallet and Desktop Wallet. Immediately contact Support by visiting this link - [store.skycoin.net/pages/support](store.skycoin.net/pages/support), or you can send an email detailing the issue to **support@skycoin.zendesk.com**.  
+It is imperative you do this as showing different addresses means your Wallets might have been compromised and using the same would lead you to the loss of your Coins.**
+
+**Wallets Showing Different Addresses**
+
+<screenshot and image of different addresses being displayed>
+
+**Step 5** - After confirming both the addresses are same, proceed to confirm the operation on your Skywallet.
```
Comment (jdknives) [inline Sending and Receiving Coins Using your Skywallet.md]: the same

Comment (jdknives) [reply Sending and Receiving Coins Using your Skywallet.md]: Confirm what? Isnt receiving possible without any confirmation?

File: Sending and Receiving Coins Using your Skywallet.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+**Step 7** - Click on the **Preview** button to get a preview of your transaction, you can once again verify all the data entered.
+
+<screenshot with the Preview button highlighted and then the screenshot of the window showing the preview of the transaction>
+
+**Step 8** - After verifying the entered data, click on the **Send** button to proceed with your transaction.
+
+<Screenshot of the Desktop Wallet with the Send button highlighted>
+
+**Step 9** - Once you click on the "Send" button, a confirmation message will be displayed on your Skywallet.
+<image of the Skywallet display screen with the confirmation message to send a certain amount of Coins to a specific address>
```
Comment (jdknives) [inline Sending and Receiving Coins Using your Skywallet.md]: Here another step needs to be entered to confirm the send on the Skywallet

**Review by jdknives — CHANGES_REQUESTED** [review]

File: Sending and Receiving Coins Using your Skywallet.md (hunk 6 above)
Comment (sreekumar13) [reply Sending and Receiving Coins Using your Skywallet.md]: > Confirm what? Isnt receiving possible without any confirmation?

This was mentioned in the existing wiki, have not personally tried it.

Comment (sreekumar13) [reply Sending and Receiving Coins Using your Skywallet.md]: Checked this, there is no confirmation message as such, just a dialogue box appearing that says, you can view the address by clicking on Show address button. Hence, will remove the second part of this sentence.

**Pull request merged**